applications/RANSApplication/custom_processes/rans_check_vector_bounds_process.h

```diff
@@ -0,0 +1,123 @@
+//    |  /           |
```

Comment:
Same comment about simplifying it to be a function. I think we can create a double and vector (maximum and minimum norm) function. The implementation can be unique if we pass a function argument with the maximum and minimum values calculation.

Reply:
removed.